src/react/components/NativeBlockWidget.tsx

```diff
@@ -19,7 +19,15 @@
 import React from 'react';
 import { DefaultPortLabel } from '@projectstorm/react-diagrams';
 
-const NativeBlockWidget = ({ engine, node, hasSettings }) => {
+import NativeBlockModel from '../models/NativeBlockModel';
+
+interface Props {
+  engine: DefaultPortLabel['props']['engine'];
```

Comment:
same as previous comment. This is also a react-diagram engine, we shouldn't rely on DefaultPortLabel for its definition